admin_manual/configuration_files/big_file_upload_configuration.rst

```diff
@@ -67,9 +67,20 @@ Apache with mod_fcgid
 ^^^^^^^^^^^^^^^^^^^^^
 * `FcgidMaxRequestLen <https://httpd.apache.org/mod_fcgid/mod/mod_fcgid.html#fcgidmaxrequestlen>`_
 
-NginX
+nginx
 ^^^^^
 * `client_max_body_size <http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size>`_
+* `fastcgi_read_timeout <http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_read_timeout>`_
+* `client_body_temp_path <http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_temp_path>`_
+
+For more info how to configure nginx to raise the upload limits see also `this
+<https://github.com/owncloud/documentation/wiki/Uploading-files-up-to-16GB#configuring-nginx>`_
+wiki entry.
+
+.. note:: For ``client_body_temp_path`` make sure it is pointing to a path/partition which holds
+   enough space for the filesize you want to upload. For optimal performance this should be also
```

Comment:
Optimal performance could be achieved in the case of the move/rename operation between client_body_temp_path and upload_tmp_dir. But as I see there is copying. The copy operation might be more efficient when partitions are placed on different devices.